torba/torba/client/basenetwork.py

```diff
@@ -24,61 +23,88 @@ def __init__(self, *args, network, server, timeout=30, **kwargs):
         self.bw_limit = self.framer.max_size = self.max_errors = 1 << 32
         self.timeout = timeout
         self.max_seconds_idle = timeout * 2
-        self.ping_task = None
+        self.latency = 1 << 32
+        self._on_connect_cb = on_connect_callback or (lambda: None)
+        self.trigger_urgent_reconnect = asyncio.Event()
+
+    @property
+    def available(self):
+        return not self.is_closing() and self._can_send.is_set() and self.latency < 1 << 32
 
     async def send_request(self, method, args=()):
         try:
-            return await asyncio.wait_for(super().send_request(method, args), timeout=self.timeout)
+            start = perf_counter()
+            result = await asyncio.wait_for(
+                super().send_request(method, args), timeout=self.timeout
+            )
+            self.latency = perf_counter() - start
```

Comment:
Can this be named something like `response_time` or `response_latency`? As to not be confusing if/when connection and round trip (time from sending a request to beginning to receive bytes from the response) latencies are added